src/build_options.rs

```diff
@@ -94,6 +94,7 @@ pub struct CargoOptions {
     pub ignore_rust_version: bool,
 
     /// Use verbose output (-vv very verbose/build.rs output)
+    // Note that this duplicates the global option, but clap seems to be fine with that.
```

Comment:
This feels wrong but unfortunately i don't understand the cargo argument handling enough to solve this properly